packages/compiler/src/core/parser.ts

```diff
@@ -2435,6 +2444,21 @@ function createParser(code: string | SourceFile, options: ParseOptions = {}): Pa
     };
   }
 
+  /**
+   * Handles the optional hyphen in param-like comment tags.
+   *
+   * The TSDoc spec expects a hyphen in the `@param` case, but it does not
+   * require it in practice. Thus, TypeSpec should handle both cases.
+   */
+  function parseParamLikeHyphen() {
```

Comment:
```suggestion
  function parseOptionalParamLikeHyphen() {
```

Reply:
Thanks, with my rewording I made it `parseOptionalHyphenDocParamLikeTag` for parallel naming.